src/scripts/main.js

```diff
@@ -1 +1,84 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
```

Comment:
The rejection message should be a string instead of an Error object to match the resolution message type. Consider using a string for consistency.